common-auth/SECURITY.md

```diff
@@ -0,0 +1,65 @@
+# Running registry and streamline web-services securely
+
+This module is intended to be used by registry and streamline web-services so that they can enable http client authentication via SPNEGO. Currently supported 
+authentication mechanism is Kerberos. The code for this module has been borrowed from hadoop-auth(2.7.3) module in hadoop project and changed a little. The 
+reasons for doing so are to avoid having a dependency on hadoop-auth module which brings in some other modules, avoid conflicts with other versions of 
+hadoop-auth module and having more control over the changes needed in future. Some text for this document has been borrowed from SECURITY.md of Apache Storm  
+
+By default, registry and streamline web-services are running without authentication enabled and anyone can access the web-services from ui/client as far as they
+know the url and can access the web-server from the client machine. To enable authentication with the client, webservice needs to add a servlet filter from 
+this module. The webservice module will need to declare a dependency on this module. One way of adding a servlet filter in code is as follows. 
+
```

Comment:
I am not sure if i understand correctly what the above means and if we should add that. May be i am missing something here.